### PR TITLE
Disable LambdaDevServicesContinuousTestingTestCase

### PR DESCRIPTION
**Pull request by gsmet**
See https://github.com/quarkusio/quarkus/issues/33963

### CODE REVIEW

**Review by geoand — APPROVED** [review]

File: extensions/amazon-lambda/deployment/src/test/java/io/quarkus/amazon/lambda/deployment/testing/LambdaDevServicesContinuousTestingTestCase.java [hunk 1]
```diff
@@ -6,12 +6,14 @@
...
 import org.jboss.shrinkwrap.api.spec.JavaArchive;
 import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Disabled;
 import org.junit.jupiter.api.RepeatedTest;
 import org.junit.jupiter.api.extension.RegisterExtension;
 
 import io.quarkus.test.ContinuousTestingTestUtils;
 import io.quarkus.test.QuarkusDevModeTest;
 
+@Disabled
```
Comment (machi1990) [inline extensions/amazon-lambda/deployment/src/test/java/io/quarkus/amazon/lambda/deployment/testing/LambdaDevServicesContinuousTestingTestCase.java:16]: Do we need to add a link to the Github issue as the reason?

**Review by machi1990 — APPROVED** [review]
lgtm

left a small question/suggestion

File: extensions/amazon-lambda/deployment/src/test/java/io/quarkus/amazon/lambda/deployment/testing/LambdaDevServicesContinuousTestingTestCase.java (hunk 1 above)
Comment (gsmet) [reply extensions/amazon-lambda/deployment/src/test/java/io/quarkus/amazon/lambda/deployment/testing/LambdaDevServicesContinuousTestingTestCase.java:16]: done!

**Review by machi1990 — APPROVED** [review]
thanks @gsmet

**Pull request merged**